### DISCUSSION TITLE
I can'not see full ros topic in host PC

### DISCUSSION THREAD

**Discussion by orangelee89** [iRobotEducation/create3_docs#606] [Questions]
### How are you connecting to your Create 3?

Wi-Fi and USB Ethernet (ROS 2)

### Computer(s) Model(s) and Operating System(s)

ubuntu 20.04   

### Which version of ROS 2 is installed on your computer?

Galactic

### Which firmware version is installed on your robot?

 G.5.4

### Which RMW is your robot running?

CycloneDDS

### Is the robot connected to a network? If so, what is the network type?

no

### Are there multiple Create 3 robots connected to your network?

No

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

USB

### Describe your question.

I can'not see full ros topic in host PC，here is what I can see in host pc. But in raspberry pi , I'm able to see all the topic list like /cmd_vel etc. (should icreate 3 connect with 2.4 Ghz wifi also ? In this case the icreate didn't connect wifi  )

$ ros2 topic list
/battery_state
/color/preview/image
/diagnostics
/diagnostics_agg
/diagnostics_toplevel_state
/dock
/hazard_detection
/imu
/interface_buttons
/ip
/joint_states
/joy
/mouse
/parameter_events
/robot_description
/rosout
/scan
/stereo/depth
/tf
/tf_static
/wheel_status

And the raspberry pi and PC are in same network:
And the raspberry pi and PC are in same network:

raspberry pi :
usb0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST> mtu 1500
inet 192.168.186.3 netmask 255.255.255.0 broadcast 192.168.186.255

wlan0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST> mtu 1500
inet 192.168.0.103 netmask 255.255.255.0 broadcast 192.168.0.255

PC:
wlo1: flags=4163<UP,BROADCAST,RUNNING,MULTICAST> mtu 1500
inet 192.168.0.101 netmask 255.255.255.0 broadcast 192.168.0.255

**Comment by alsora** [comment]
Hi @orangelee89,
i think that all the topics that you see are from the Pi and not from the Create 3. i.e. you are not seeing any create 3 topic at all.
Can you do this to confirm it?
```
ros2 topic list --spin-time 10 -v
```

> (should icreate 3 connect with 2.4 Ghz wifi also ? In this case the icreate didn't connect wifi )

Yes, you need the create 3 to be connected to the Wifi to see its topics from the pc.
